webapp/src/tutorial.tsx

```diff
@@ -286,7 +291,7 @@ export class TutorialComplete extends data.Component<ISettingsProps, TutorialCom
                 closeOnDimmerClick closeOnDocumentClick
                 >
                 <div className="ui two stackable cards">
-                    <div className="ui grid centered link card focused" aria-selected="true" aria-label={lf("More Tutorials")} tabIndex={0} onClick={() => this.moreTutorials() } onKeyPress={sui.fireClickOnEnter}>
+                    <div className="ui grid centered link card focused" aria-selected="true" aria-label={lf("More Tutorials")} tabIndex={0} onClick={() => this.moreTutorials() } onKeyDown={sui.fireClickOnEnter}>
```

Comment:
Any other instances of this change? Could you verify there aren't any other places where we should be using onKeyDown instead of onKeyPress.

Reply:
Please also document in your dev docs, when to use which.

Reply:
Yes, there was another instance in webapp/src/app.tsx that is already included to this PR. I checked if any other onKeyPress was existing.

It is also already documented in my PR related to dev documentation : https://github.com/Microsoft/pxt/pull/2853/files#diff-32b935a28c228bd912fbdb42b89f1548 .
I quote : _"You will also have to decide on which keyboard event to use: ``KeyDown``, ``KeyUp`` or ``KeyPress``. When possible, always use the ``KeyDown`` event as ``KeyPress`` has compatibility problems with Internet Explorer. "_